src/data/services/LmsApiService.js

```diff
@@ -0,0 +1,24 @@
+import qs from 'query-string';
+
+import apiClient from '../apiClient';
+
+class LmsApiService {
+  static UpdateEmailSettings(courseRunId, hasEmailsEnabled) {
+    const queryParams = {
+      course_id: courseRunId,
+      email_opt_in: hasEmailsEnabled,
+    };
+    const emailSettingsUrl = `${apiClient.authBaseUrl}/user_api/v1/preferences/email_opt_in/`;
```

Comment:
I think we should use the actual environment variable here instead of `apiClient.authBaseUrl` to be more explicit and less reliant on the `apiClient`: `process.env.LMS_BASE_URL`.

Reply:
agreed, this didn't feel right :)